src/components/RegionSelect.vue

```diff
@@ -78,11 +92,100 @@ export default {
       type: Boolean,
       default: false,
     },
+    removePlaceholder: {
+      type: Boolean,
+      default: false
+    },
+    usei18n: {
+      type: Boolean,
+      default: true,
+    },
+    autocomplete: {
+      type: Boolean,
+      default: false,
+    }
+  },
+  data: () => ({
+    shownRegions: [],
+    regions,
+    ran: false
+  }),
+  mounted() {
+    if (this.country) {
+      this.getRegionWithCountry()
+    } else {
+      let findRegion = ''
+      if (this.countryName) {
+        findRegion = this.defaultRegion ? this.defaultRegion : 'United States'
+      } else {
+        findRegion = this.defaultRegion ? this.defaultRegion : 'US'
```

Comment:
I would think the fallback region would be Canada, no?

Reply:
Ah good catch, that was leftover from VueRegionSelect